mysql-test/suite/galera/t/galera_autoinc_sst_mariabackup.test

```diff
@@ -0,0 +1,96 @@
+#
+# Test that autoincrement works correctly while the cluster membership
+# is changing and SST takes place.
+#
+
+--source include/big_test.inc
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+--source include/have_mariabackup.inc
+
+--connection node_1
```

Comment:
I suggest that you add here:

--let $node_1=node_1
--let $node_2=node_2
--source include/auto_increment_offset_save.inc

Problem is that you do not always know what order servers come up